aries_cloudagent/resolver/default/indy.py

```diff
@@ -43,6 +46,60 @@ def supported_did_regex(self) -> Pattern:
         """Return supported_did_regex of Indy DID Resolver."""
         return IndyDID.PATTERN
 
+    def _add_endpoint_as_endpoint_value_pair(
+        self,
+        builder: DIDDocumentBuilder,
+        endpoint: str,
+        recipient_key: VerificationMethod,
+    ):
+        builder.service.add_didcomm(
+            ident=self.SERVICE_TYPE_DID_COMMUNICATION,
+            type_=self.SERVICE_TYPE_DID_COMMUNICATION,
+            service_endpoint=endpoint,
+            priority=1,
+            recipient_keys=[recipient_key],
+            routing_keys=[],
+        )
+
+    def _add_endpoint_as_map(
+        self,
+        builder: DIDDocumentBuilder,
+        endpoint: Mapping[str, Any],
+        recipient_key: VerificationMethod,
+    ):
+        types = endpoint.get("types", [self.SERVICE_TYPE_DID_COMMUNICATION])
+        routing_keys = endpoint.get("routingKeys", [])
+        endpoint_url = endpoint.get("endpoint")
+        if not endpoint_url:
+            raise ValueError("endpoint url not found in endpoint attrib")
+
+        if self.SERVICE_TYPE_DIDCOMM in types:
+            builder.service.add(
+                ident="#didcomm-1",
+                type_=self.SERVICE_TYPE_DIDCOMM,
+                service_endpoint=endpoint_url,
+                recipient_keys=[recipient_key.id],
+                routing_keys=routing_keys,
+                accept=["didcomm/v2"],
```

Comment:
Isn't it an issue that it resolves to accept only didcomm/v2, while we don't actually support that yet? Or does it not matter because this endpoint isn't examined anyway.

Reply:
Yeah, I think in this case, we're just constructing the DID Document as the DID Method spec outlines and whether we use the all of the services outlined in the doc is another question. We will be essentially ignoring them for now since we're only looking for the `did-communication` type.